### DISCUSSION TITLE
directus_users -> tags docs data type inconsistency

### DISCUSSION THREAD

**Discussion by anlexN** [directus/directus#15624] [Archive]
![image](https://user-images.githubusercontent.com/8686739/190845506-f8e49b40-4b33-4a4c-ad96-59854f8b9698.png)

![image](https://user-images.githubusercontent.com/8686739/190845553-5c3baa51-560d-4633-8df4-ad93fa6c6196.png)

I use postgresql

**Comment by azrikahar** [comment]
I believe the docs is outdated in this case. Mind opening a PR to change `csv` to `array` just like how `directus_files` `tags` is: https://github.com/directus/docs/blob/05ac73bcd35b423f95d4fcc19149ee88815be9b8/reference/files.md?plain=1#L203-L204? Thanks!

**Comment by anlexN** [comment]
I have [opened pull request](https://github.com/directus/docs/pull/147), please merge.
